lib/rails_response_dumper/runner.rb

```diff
@@ -57,7 +57,15 @@ def run_dumps
             else
               extension = ''
             end
-            File.write("#{dumper_dir}/#{index}#{extension}", response.body)
+
+            File.open("#{dumper_dir}/#{index}#{extension}", 'w') do |f|
+              f.write "HTTP #{response.status} #{response.status_message}\r\n"
```

Comment:
Could this use `f.puts` to write the new line instead?  

https://ruby-doc.org/core-2.0.0/IO.html#method-i-puts

Reply:
That would not include a carriage return. It would be a line feed only.